tools/ci_connector_ops/ci_connector_ops/qa_checks.py

```diff
@@ -20,16 +20,16 @@ def check_migration_guide(connector: Connector) -> bool:
 
     migration_guide_file_path = connector.migration_guide_file_path
     if not migration_guide_file_path.exists():
-        print(f"Migration guide file is missing for {connector.name}.")
+        print(f"Migration guide file is missing for {connector.name}. Please create a {connector.migration_guide_file_name} file in the docs folder.")
         return False
 
     # Check that the migration guide begins with # {connector name} Migration Guide
-    expected_title = f"# {connector.name} Migration Guide"
+    expected_title = f"# {connector.name_from_metadata} Migration Guide"
```

Comment:
Change: we use the name (title case, official name) from the metadata, not the `source-<whatever>` name in the migration guide title